lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.ts

```diff
@@ -219,13 +224,20 @@ export class TaskListCloudComponent extends DataTableSchema implements OnChanges
         return this.currentInstanceId;
     }
 
-    private isPropertyChanged(changes: SimpleChanges): boolean {
+    private isAnyPropertyChanged(changes: SimpleChanges): boolean {
         for (const property in changes) {
-            if (changes.hasOwnProperty(property)) {
-                if (changes[property] &&
-                    (changes[property].currentValue !== changes[property].previousValue)) {
-                    return true;
-                }
+            if (this.isPropertyChanged(changes, property)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private isPropertyChanged(changes: SimpleChanges, property: string): boolean {
```

Comment:
if you get a property in the SimpleChanges object, that already means it has been changed, so I guess no need validating that once again. see details: https://angular.io/api/core/OnChanges

Reply:
Updated